x-pack/platform/packages/private/ml/aiops_components/src/progress_controls/progress_controls.tsx

```diff
@@ -139,7 +139,7 @@ export const ProgressControls: FC<PropsWithChildren<ProgressControlProps>> = (pr
         {progress === 1 ? (
           <EuiFlexGroup gutterSize="xs" alignItems="center" responsive={false}>
             <EuiFlexItem grow={false}>
-              <EuiIcon type="checkInCircleFilled" color={euiTheme.colors.success} />
+              <EuiIcon type="checkInCircleFilled" color={euiTheme.colors.accentSecondary} />
```

Comment:
I wonder if this should use 'success', according to:
> If conveying some affirmative meaning - success, a positive action, etc. -- continue using the success color
> If not -- please update the color to accentSecondary

![image](https://github.com/user-attachments/assets/d79c9d09-2fdf-49f3-bbc8-4bf26fe277fd)
In this scenario, it actually has `an affirmative meaning`, if I’m correct. However, using `accentSecondary` aligns it with the badge on the right side, so perhaps it’s better to stick with `accentSecondary`.